woocommerce/woocommerce
# CYS: Remove spotlight tour modal shadow

### Changes proposed in this Pull Request:

Closes #40819.

Updates CSS to match spotlight tour modal design by removing drop shadow and adding border.

### How to test the changes in this Pull Request:

1. Make sure to enable `customize-store` feature flag
1. Go to `WooCommerce -> Home` and click `Customize Store` task
1. Go through the AI wizard and proceed to the assembler hub
2. If Tour modal doesn't show, you'll need to delete the site option `woocommerce_customize_store_onboarding_tour_hidden`
1. When tour modal shows, click on `Take a tour`
3. Observe the tour modal doesn't have any shadow and matches design (Mrk6SERPZ4KrFHSjM0a8TK-fi-3529%3A279144)
4. Click on `Next`
3. Observe the tour modal doesn't have any shadow and matches design (Mrk6SERPZ4KrFHSjM0a8TK-fi-3529%3A279518)

<img width="354" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/c499409d-c104-40b0-8967-9fd01c8ccb1a">

<img width="343" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/97f3566d-79ee-4cfb-b0b4-8aeeba9c212b">





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
